static/src/javascripts/projects/commercial/modules/article-body-adverts.ts

```diff
@@ -70,7 +70,7 @@ const addDesktopInlineAds = (isInline1: boolean): Promise<boolean> => {
 	const isImmersive = config.get('page.isImmersive');
 	const defaultRules: SpacefinderRules = {
 		bodySelector: articleBodySelector,
-		slotSelector: ' > p',
+		slotSelector: ['p', 'ul li'],
```

Comment:
It seems like we could make `slotSelector` a bit more precise if there there was a way to distinguish between the two article types, those made up entirely of bullet points (I'm sure there's a real name for these) like the ones you've linked to and normal articles made up of mostly paragraphs. This way we could select just the relevant elements for the current article type.
This would also ensure that lists in "normal" articles to behave differently from lists in "bullet point" articles (assuming that this is the behaviour we want).
Maybe something that dotcom folks would know more about?

Reply:
Thanks for the review @zekehuntergreen 😄 

I might have put too much emphasis on articles which are entirely lists in the PR description. I feel this solution would be equally appropriate for articles which are mostly paragraphs but also contain lists, for example:

https://www.theguardian.com/environment/2022/feb/28/ipcc-issues-bleakest-warning-yet-impacts-climate-breakdown?sfdebug=1

In both cases it feels like an oversight that we're not considering lists for ad insertion.

Very happy to be overruled if there's some distinction I'm missing between the two cases!

Reply:
Ah ok that makes sense - I was thinking that the answer to your question around whether or not we are willing to break up lists on mobile might depend on the type of article / how long the list is e.g. if the list takes up the whole article, we break it up with inline ads, if its shorter, we may not want to. But personally I don't think I have a strong opinion on this - it may be a question for the wider team + dotcom?